Pod/Classes/WPMediaCollectionViewController.m

```diff
@@ -209,22 +250,14 @@ - (void)refreshTitle {
     } else {
         self.titleButton.hidden = NO;
     }
-    NSString *localizedOptionHint = NSLocalizedString(@"Tap here to change", "Tip for tapping media picker title to change the group.");
     NSString *albumName = NSLocalizedString(@"No Photos", "Group name to show when permission are denied to access Photos albums");
     if ([mediaGroup name] != nil) {
         albumName = [mediaGroup name];
     }
-    NSString *albumLine = [NSString stringWithFormat:@"%@\n", albumName];
     UIFont *titleFont = self.titleButton.titleLabel.font;
-    NSMutableAttributedString *title = [[NSMutableAttributedString alloc] initWithString:albumLine attributes:@{NSFontAttributeName: titleFont}];
-    NSString *callForAction = [NSString stringWithFormat:@"%@ %@",localizedOptionHint, ArrowDown];
-    [title appendAttributedString:[[NSAttributedString alloc] initWithString:callForAction attributes:@{NSFontAttributeName: [titleFont fontWithSize:floorf(titleFont.pointSize * 0.75)]}]];
+    NSMutableAttributedString *title = [[NSMutableAttributedString alloc] initWithString:albumName attributes:@{NSFontAttributeName: titleFont}];
```

Comment:
Can we just set the font of the button's titleLabel instead of creating an attributed string?